test_unstructured/partition/test_csv.py

```diff
@@ -22,6 +23,7 @@ def test_partition_csv_from_file(filename="example-docs/stanley-cups.csv"):
     assert isinstance(elements[0], Table)
     assert elements[0].metadata.text_as_html == EXPECTED_TABLE
     assert elements[0].metadata.filetype == EXPECTED_FILETYPE
+    # assert elements[0].metadata.filename == "stanley-cups.csv"
```

Comment:
Should these asserts be commented out?

Reply:
I see, this is the `from_file` test. In that case, the expected behavior is for `filename` to be `None` since no filename is provided (unless you pass `metadata_filename`). This one (and any other `from_file` tests without `metadata_filename` passed) you can change to `assert elements[0].metadata.filename is None`